build.gradle

```diff
@@ -0,0 +1,32 @@
+group 'org.apache.cassandra'
+version '1.0'
+
+apply plugin: 'java'
+apply plugin: 'application'
+
+sourceCompatibility = 1.8
+
+repositories {
+    maven { url "http://172.16.20.36:8081/repository/maven-public/" }
```

Comment:
Can you remove this maven repo? You should be able to configure private repos in your maven settings.xml, and that will mean you don't need to include them in your build files.